scss/components/_media-object.scss

```diff
@@ -50,6 +62,27 @@ $mediaobject-image-width-stacked: 100% !default;
   > :last-child {
     margin-bottom: 0;
   }
+
+  .stack-for-#{$-zf-zero-breakpoint} & {
```

Comment:
I agree the code would be more cleaner with this in `media-object-selection`. But sadly, `.stack-for-medium .media-object-section` is not the same as `.media-object.stack-for-medium .media-object-section`, bugs can be introduced because `.stack-for-*` is used for other components.

With this selector, if any parent use `stack-for-` for its own purpose, this component will be affected. So i'm moving this to `@mixin media-object-container` to ensure backward compatibility.

(I don't like it neither, it prevent to reduce the specificity 😢  ).